docs/source/user_guide/cuspatial_api_examples.ipynb

```diff
@@ -133,7 +133,8 @@
     "import cudf\n",
```

Comment:
<p>I wonder if this is a change that was made at the shapely level? I'm not familiar with seeing the polygon points when printing a shapely object.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/rapidsai/cuspatial/pull/906/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='16'/>